### PR TITLE
Add new settings ``RACK_LISTING_NUMBERING_TOP_TO_BOTTOM``

### PR DESCRIPTION
**Pull request by ar4s**
Set `RACK_LISTING_NUMBERING_TOP_TO_BOTTOM = True` for reverse number ordering in rack
visualization.

```
RACK_LISTING_NUMBERING_TOP_TO_BOTTOM = True   # 1..max_u_height from top to bottom
RACK_LISTING_NUMBERING_TOP_TO_BOTTOM = False  # max_u_height..1 from top to bottom
```

image

### CODE REVIEW

**Comment by ar4s** [comment]
image

File: src/ralph/dc_view/templates/dc_view/settings_for_angular.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% load cache staticfiles %}
+
+{% cache 3600 settings_js %}
```
Comment (mkurek) [inline src/ralph/dc_view/templates/dc_view/settings_for_angular.js]: Do we have to cache it?

Comment (ar4s) [reply src/ralph/dc_view/templates/dc_view/settings_for_angular.js]: nope :)

**Review by ghost — APPROVED** [review]

File: src/ralph/settings/base.py [hunk 2]
```diff
@@ -339,6 +339,12 @@ def str_to_bool(s: str) -> bool:
 }
 
 # =============================================================================
+# DC view
+# =============================================================================
+
+RACK_LISTING_REVERSE = True
```
Comment (mkurek) [inline src/ralph/settings/base.py]: default should be False. And consider changing name of this variable to better show numeration (from top to bottom or from bottom to top).

**Review by mkurek — CHANGES_REQUESTED** [review]

File: src/ralph/settings/base.py (hunk 2 above)
Comment (ar4s) [reply src/ralph/settings/base.py]: done

**Review by mkurek — APPROVED** [review]

**Pull request merged**